### PR TITLE
Added error testing console and updated VC.Module dependency

### PR DESCRIPTION
**Pull request by pushnitsa**

### CODE REVIEW

File: Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs [hunk 1]
```diff
@@ -212,8 +217,10 @@ public override PostProcessPaymentResult PostProcessPayment(PostProcessPaymentEv
             }
             else
             {
+                const int defaultErrorCode = -1000;
```
Comment (t13ka) [inline Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs]: what means this magic number -1000?

Comment (pushnitsa) [reply Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs]: Name of it, not enough meaningful?

Comment (t13ka) [reply Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs]: Actually no. why -1000? Is there have a description of all error codes anywhere?
I guess should be better if we place these constants anywhere with reference to documentation

Comment (pushnitsa) [reply Datatrans.Checkout/Managers/DatatransCheckoutPaymentMethod.cs]: You suggest place all error codes just only for description? We need a default error code, which does not described in documentation, that's why -1000

**Comment by yecli** [comment]
Please address sonar warnings.
For potential password loss - seems it should be suppressed.

**Pull request merged**